docs/pages/Workspace.md

```diff
@@ -0,0 +1,125 @@
+# UI5 Workspaces
+
+!!! warning
+    **UI5 Workspaces is a feature currently in development.** It is not part of any UI5 Tooling release and therefore cannot be used yet.  
+
+!!! example
+    ```yaml title="ui5-workspace.yaml"
+    specVersion: workspace/1.0
+    metadata:
+        name: default
+    dependencyManagement:
+   	    resolutions:
+       	    - path: ../heavy.library
+    ```
+
+## General Concept
+
+UI5 Workspaces can be used to create a personalized local development environment for a UI5 Project. They allow to use UI5 dependencies from local directories without the need to use the link features of package managers like npm. "UI5 dependencies" generally refers to projects that have a `ui5.yaml`.
+
+Workspaces are typically configured in a `ui5-workspace.yaml` file, located next to the project's `ui5.yaml`. The file can contain one or many workspace configurations, each separated by [three dashes](https://yaml.org/spec/1.2.2/#22-structures).
+
+If there is a workspace configuration named `default`, it will be used automatically. Otherwise the workspace to use needs to be specified using the UI5 CLI parameter `--workspace`.
```

Comment:
```suggestion
If a workspace configuration named `default` exists, it will be used automatically; otherwise the workspace must be specified using the UI5 CLI parameter `--workspace`.
```